custom_components/tuya_local/devices/tuya_wifi_energy_consumption_meter.yaml

```diff
@@ -0,0 +1,157 @@
+name: Tuya Wifi Energy Consumption Meter
```

Comment:
Drop the "Tuya Wifi", as this is redundant for an integration that only supports Tuya devices.  The device name is prepended to all entity names, so should be reasonably short.  Maybe even "Energy meter" is enough.

Also, device and entity names should follow [Home Assistant naming guidelines](https://developers.home-assistant.io/blog/2022/07/10/entity_naming/) in which only the first word is capitalised unless a brand name or abbreviation.  This also applies to the "Total energy" sensors below (but the A and B can be considered abbreviations that should be capitalised).